tests/filters/fbuffer.py

```diff
@@ -0,0 +1,343 @@
+import os
+import uuid
+import json
+import redis
+import logging
+from time import sleep, time
+
+from tools.redis_utils import RedisServer
+
+from tools.filter import Filter
+from tools.output import print_result
+from darwin import DarwinApi, DarwinPacket
+
+REDIS_SOCKET = "/tmp/redis_buffer.sock"
+ALERT_FILE = "/tmp/test_fbuffer.txt"
+FILTER_CODE = 0x62756672
+
+class Buffer(Filter):
+    def __init__(self):
+        super().__init__(filter_name="buffer", cache_size=1, path="../build/darwin_buffer")
```

Comment:
don't override **path**, prefer changing **DEFAULT_FILTER_PATH** in your test/conf.py instead